composer.json

```diff
@@ -0,0 +1,15 @@
+{
+    "name": "ipman/premier_blog",
+    "description": "Mon premier blog",
+    "type": "project",
+    "require": {
+        "twig/twig": "^3.3"
+    },
+    "authors": [
+        {
+            "name": "Nicolas Yang",
+            "email": "nicolas.yang.1@gmail.com"
+        }
+    ],
+    "minimum-stability": "stable"
+}
```

Comment:
I think you should add an src folder to contain all php code not accessible in public.
And add this follow config to have PSR4 compliance with your autoloader.

    "autoload": {
        "psr-4": {
            "App\\": "src/"
        }
    },